CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [4.0.4]
+### Added
+- Support for new PayPal integration.
```

Comment:
This is not support for PayPal re integration only is it? We are also changing the way we are handling the mismatch in order lines amounts, to be in line to what would be expected by many acquirers.

```suggestion
### Fixed
- Streamline and simplify management of order lines and amounts.
- Support for the new PayPal Integration.
```